test-service/README.adoc

```diff
@@ -0,0 +1,46 @@
+= Kiali Test Mesh
+:toc: macro
+:toc-title:
+
+toc::[]
+
+= Introduction
+
+Kiali Traffic Generator is a component which generate then ecessary traffic for visualization of Kiali Test Meshes on Kiali Service Graph.
+
+It uses https://github.com/tsenart/vegeta[Vegeta], a lightweight golang traffic generator.
+
+== Deploying in OpenShift
+
+oc process -f https://raw.githubusercontent.com/kiali/kiali-test-mesh/master/test-service/traffic-generator/openshift/traffic-generator.yaml -p ROUTE=your_route_a_microservice/route?path=your_mesh_example -e  DURATION="0s" -e RATE="50" | oc apply -n namespace  -f -
+
+=== ROUTE parameter
+your by mesh example, it can assume following values:
+
+- `a,b,c,d,e,f` (all together) for Depth
+- `a` `b` `c` `d` `e` `f` (one query per letter) for Breath
+- `a,b,c,d,e,f,a` for Circle
+- `a,b,c,d,e,f,a` ; `a,f,e,d,c,b,a` for Cicle Back
+- `a,b,c,d,e` ; `a,c,e` for Hourglass
+- `a,b,d,c,a,d,c,b` for Box
+- `a,f`;`a,b,f`;`a,b,c,f`;`a,b,c,d,f`;`a,b,c,d,e,f` for Depth Sink
+- `a,f`;`b,f`;`c,f`;`d,f`;`a,b,c,d,e,f` for Breadth Sink
+
+Important:
+
+- *You may use mutiple routes. They must be separate by ; (semicolon)*
+`eg: http://a-kiali-test-breath.your.openshift.com/route?path=a;http://a-kiali-test-breath.your.openshift.com/route?path=b`
+
```

Comment:
The urls here are wrong, instead of 'a-kiali-test-breath.your.openshift.com' just use 'a', the dns system used by OpenShift/k8s will resolve this to a service which is running in the same project.